includes/html/pages/device/edit/device.inc.php

```diff
@@ -180,9 +180,9 @@
         </div>
     </div>
     <div class="form-group">
-        <label for="disabled" class="col-sm-2 control-label">Disable:</label>
+        <label for="disabled" class="col-sm-2 control-label">Disable polling:</label>
         <div class="col-sm-6">
-          <input name="disabled" type="checkbox" id="disabled" value="1" data-size="small"
+          <input onclick="edit.ignore.disabled=edit.disabled.checked" name="disabled" type="checkbox" id="disabled" value="1" data-size="small"
```

Comment:
your js here is not functioning... If you pull it out of this PR and create a new one once you have it work, we can mere your wording changes now.

Reply:
Yes, sorry about that. The code works in LibreNMS 1.56 but effectively doesn't work in master...